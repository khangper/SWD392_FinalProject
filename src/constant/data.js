export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
