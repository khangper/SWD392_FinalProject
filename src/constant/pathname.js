export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
