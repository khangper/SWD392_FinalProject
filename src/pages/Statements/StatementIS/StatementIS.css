.statementis {
    padding: 20px;
    background-color: #f5f5f5;
  }
  
  .statementis-row {
    display: flex;
    flex-wrap: wrap;
    margin-bottom: 20px;
  }
  
  .statementis-col-lg-12 {
    flex: 0 0 100%;
    max-width: 100%;
  }
  
  .statementis-col-lg-8 {
    flex: 0 0 66.66667%;
    max-width: 66.66667%;
    padding-right: 15px;
  }
  
  .statementis-col-lg-4 {
    flex: 0 0 33.33333%;
    max-width: 33.33333%;
    padding-left: 15px;
  }
  
  .st_title {
    font-size: 24px;
    font-weight: bold;
    display: flex;
    align-items: center;
  }
  
  .st_title i {
    margin-right: 10px;
  }
  
  .top_countries {
    background-color: #fff;
    border-radius: 8px;
    padding: 20px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
  }
  
  .top_countries_title {
    border-bottom: 1px solid #eaeaea;
    padding-bottom: 10px;
    margin-bottom: 20px;
  }
  
  .top_countries_title h2 {
    font-size: 18px;
    font-weight: bold;
  }
  
  .statement_content .tt-body {
    font-weight: 400 !important;
    font-size: 14px;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 15px;
    line-height: 26px;
    color: #686f7a !important;
    text-align: left;
  }
  
  .apply_instructor_btn {
    display: inline-block;
    height: 40px;
    font-size: 14px;
    font-weight: 500;
    color: #fff;
    background: #ed2a26;
    font-family: 'Roboto', sans-serif;
    padding: 10px 30px;
    border-radius: 3px;
    margin-top: 14px;
  }
  
  .statement_invoice_content {
    text-align: center;
  }
  
  .date_selector {
    display: flex;
    align-items: center;
    justify-content: space-between;
  }
  
  .ui.dropdown.skills-search {
    width: 70%;
  }
  
  .st_download_btn {
    background-color: #f44336;
    color: #fff;
    border: none;
    padding: 10px;
    border-radius: 5px;
    cursor: pointer;
  }
  
  .more_options_tt {
    list-style: none;
    padding: 0;
    padding-top: 50px;
    display: flex;
    align-items: center;
  }
  
  .more_options_tt li {
    width: auto;
    margin-right: 10px;
  }
  
  .more_options_tt .more_items_14 {
    background-color: #f5f5f5;
    border: 1px solid #ddd;
    padding: 10px 20px;
    border-radius: 5px;
    cursor: pointer;
  }
  
  .more_options_tt .more_items_14.active {
    background-color: #333;
    color: #fff;
  }
  
  .explore_search {
    flex: 1;
  }
  
  .ui.search .input {
    width: 100%;
  }
  
  .sis-box-iput {
    width: 100%;
  }
  
  .sis-box-iput .prompt {
    border: 1px solid #ddd;
    padding: 10px;
    border-radius: 5px;
  }
  
  .sis-box-iput .icon {
    color: #ddd;
  }
  
  .sis-box-iput .icon.icon8 {
    right: 10px;
    top: 50%;
    transform: translateY(-50%);
  }
  
  .sis-search {
    background: #fff !important;
    border: 1px solid #e5e5e5 !important;
    color: #333 !important;
    box-shadow: inset 0px 1px 4px rgba(0, 0, 0, 0.02) !important;
    padding: 10px;
    border-radius: 5px;
    width: 100%;
  }

  .ui.selection {
    position: relative;
    display: inline-block;
    width: 50rem !important;
    cursor: pointer;
    padding: 10px;
    border: 1px solid #ddd;
    border-radius: 4px;
  }
  
  .section-title-wrapper {
    display: flex;
    align-items: center;
    justify-content: space-between;
  }
  
  .section-title-text {
    margin-left: 5px;
    margin-right: 15px;
  }
  
  .dropdown-menu {
    position: absolute;
    top: 100%;
    left: 0;
    width: 100%;
    background-color: #fff;
    border: 1px solid #ddd;
    border-radius: 4px;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
    z-index: 1000;
    transition: max-height 0.3s ease-in-out, opacity 0.3s ease-in-out;
  }
  
  .dropdown-menu .top {
    padding: 10px;
    cursor: pointer;
  }
  
  .dropdown-menu .top:hover {
    background-color: #f5f5f5;
  }

.statement-summary__table {
  margin: 0px auto 0px;
  font-size: 14px;
  width: 100%;
}
  
.t-heading {
  font-weight: 400 !important;
    font-size: 14px;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 15px;
    line-height: 26px;
    color: #686f7a !important;
    text-align: left;
}

.t-currency {
  font-size: 20px;
  font-weight: 600;
  font-family: 'Roboto', sans-serif;
  line-height: 24px;
  margin-bottom: 10px !important;
}

.tt__earning {
  font-size: 20px;
  font-weight: 600;
  font-family: 'Roboto', sans-serif;
  margin-right: 5px;
}

.js-earnings__earnings-wrapper {
  color: #82b541;
}

.js-earnings__fees-wrapper {
  color: #ed2a26;
}