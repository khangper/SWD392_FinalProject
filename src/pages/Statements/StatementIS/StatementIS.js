import React, { useState, useEffect } from 'react';
import './StatementIS.css'
import dropdown from "../../../assets/dropdown.png";
import { useDispatch, useSelector } from "react-redux";
import { fetchStatementISRequest } from '../../../redux/reduxActions/StatementISAction';
const StatementIS = () => {
    const dispatch = useDispatch();
    const { statementIS } = useSelector((state) => state.statementIS);

    useEffect(() => {
        dispatch(fetchStatementISRequest());
    }, [dispatch]);

    const [isOpen, setIsOpen] = useState(false);
    const [selectedValue, setSelectedValue] = useState('Monthly Invoices');

    const toggleDropdown = () => setIsOpen(!isOpen);
    const handleSelect = (value) => {
        setSelectedValue(value);
        setIsOpen(false);
    };

    const courses = [
        {
            id: 1,
            date: '13 Apr 2020',
            orderID: '123456',
            type: 'Buy',
            title: 'Course Title Here',
            amount: '$17',
            fee: '-$5'
        },
        {
            id: 2,
            date: '13 Apr 2020',
            orderID: '123456',
            type: 'Buy',
            title: 'Course Title Here',
            amount: '$17',
            fee: '-$5'
        },
        {
            id: 3,
            date: '13 Apr 2020',
            orderID: '123456',
            type: 'Buy',
            title: 'Course Title Here',
            amount: '$17',
            fee: '-$5'
        },

    ];

    return (
        <div className='statementis'>
            <div className="statementis-row">
                <div className="statementis-col-lg-12">
                    <h2 className="st_title"><i className="uil uil-file-alt"></i> Statements</h2>
                </div>
            </div>
            <div className="statementis-row">
                <div className="statementis-col-lg-8">
                    <div className="top_countries mt-30">
                        <div className="top_countries_title">
                            <h2>Earnings</h2>
                        </div>
                        <div className="statement_content">
                            <p className="tt-body">If you are not an instrutor, you can't use this section.</p>
                            <div>
                                <table className="statement-summary__table">
                                    <thead>
                                        <tr>
                                            <th>
                                                <p className="t-heading">My funds</p>
                                            </th>
                                            <th>
                                                <p className="t-heading">Earnings</p>
                                            </th>
                                            <th>
                                                <p className="t-heading">Cursus Fees</p>
                                            </th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td className="statement-summary__funds">
                                                <p className="js-earnings__instructor-funds-wrapper">
                                                    <span className=""></span>
                                                    <span className="js-earnngs__instructor-funds t-currency">$289.64</span>
                                                </p>
                                            </td>
                                            <td className="statement-summary__earnings">
                                                <p className="js-earnings__earnings-wrapper">
                                                    <span className="tt__earning">+</span>
                                                    <span className="js-earnings__earnings t-currency">$458.00</span>
                                                </p>
                                            </td>
                                            <td className="statement-summary__fees">
                                                <p className="js-earnings__fees-wrapper">
                                                    <span className="tt__earning">-</span>
                                                    <span className="js-earnings__fees t-currency">$154.86</span>
                                                </p>
                                            </td>
                                        </tr>
                                    </tbody>
                                </table>
                            </div>
                        </div>
                    </div>
                </div>

                <div className="statementis-col-lg-4">
                    <div className="top_countries mt-30">
                        <div className="top_countries_title">
                            <h2>View Invoices</h2>
                        </div>
                        <div className="statement_invoice_content">
                            <div className="date_selector">
                                <div className="ui selection" tabIndex="0" onClick={toggleDropdown}>
                                    <input name="date" type="hidden" value={selectedValue} />
                                    <div className='sis-drop'>

                                        <span className="section-title-wrapper">

                                            <span className="section-title-text">{selectedValue}</span>
                                            <img className="sidebar-dropdown-caret-icon" src={dropdown} alt="Dropdown Icon" />
                                        </span>
                                    </div>
                                    {isOpen && (
                                        <div className="dropdown-menu">
                                            <div className="top" onClick={() => handleSelect('April 2020')}>
                                                <div className="title">April 2020</div>
                                            </div>
                                            <div className="top" onClick={() => handleSelect('March 2020')}>
                                                <div className="title">March 2020</div>
                                            </div>
                                        </div>
                                    )}
                                </div>
                                <button className="st_download_btn"><i className="uil uil-download-alt"></i></button>
                            </div>
                        </div>
                    </div>
                </div>
                <div className="statementis-col-lg-12">
                    <div className="more_options_tt">
                        <li>
                            <div>
                                <button className="more_items_14 active">This Month</button>
                            </div>
                        </li>
                        <li>
                            <div>
                                <button className="more_items_14">Last Month</button>
                            </div>
                        </li>
                        <li>
                            <div className="explore_search">
                                <div className="sis-search-box">
                                    <div className="sis-box-iput">
                                        <input className="sis-search" type="text" placeholder="Document Number" />
                                        <i className="uil uil-search-alt icon icon8"></i>
                                    </div>
                                </div>
                            </div>
                        </li>
                    </div>
                </div>


            </div>
            <div className="MyCourse-tab-pane active" id="my-courses-tab" role="tabpanel">
                {/* Ná»™i dung cho tab My Courses */}

                <div className="MyCouser-containerr">
                    <div className="MyCouser-table-container">
                        <table className="MyCourse-table ucp-table">
                            <thead className="MyCourse-thead-s">
                                <tr>
                                    <th className="MyCourse-text-center" scope="col">Date</th>
                                    <th className="MyCourse-text-center">Order ID</th>
                                    <th className="MyCourse-text-center" scope="col">Type</th>
                                    <th className="MyCourse-text-center" scope="col">Title</th>
                                    <th className="MyCourse-text-center" scope="col">Amount</th>
                                    <th className="MyCourse-text-center" scope="col">Fees</th>
                                    <th className="MyCourse-text-center" scope="col">Invoice</th>

                                </tr>
                            </thead>
                            <tbody>
                                {statementIS.map(course => (
                                    <tr key={course.id}>
                                        <td className="MyCourse-text-center">{course.date}</td>
                                        <td>{course.orderID}</td>
                                        <td className="MyCourse-text-center">{course.type}</td>
                                        <td className="MyCourse-text-center">{course.title}</td>
                                        <td className="MyCourse-text-center">{course.amount}</td>
                                        <td className="MyCourse-text-center">{course.fee}</td>
                                        <td className="MyCourse-text-center">
                                            <a href="#">View</a>
                                        </td>
                                    </tr>
                                ))}
                            </tbody>
                        </table>
                    </div>

                </div>

            </div>
        </div>

    )
}

export default StatementIS