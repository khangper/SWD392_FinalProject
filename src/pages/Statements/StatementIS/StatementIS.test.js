import React from "react";
import { renderWithProviders } from "../../../constant/test-util";
import StatementIS from "./StatementIS"
import { screen } from "@testing-library/react";

describe("StatementIS", () => {
    beforeAll(() => {
        jest.spyOn(console, "error").mockImplementation(() => { });
        jest.spyOn(console, "log").mockImplementation(() => { });
    });

    afterAll(() => {
        console.error.mockRestore();
        console.log.mockRestore();
    });
    it("renders the StatementIS component", async () => {
        const { container } = renderWithProviders(<StatementIS />);
        expect(container).toBeDefined();
        expect(container).toMatchSnapshot();
    });
})