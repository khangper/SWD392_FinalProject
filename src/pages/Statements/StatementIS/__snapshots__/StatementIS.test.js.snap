// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatementIS renders the StatementIS component 1`] = `
<div>
  <div
    class="statementis"
  >
    <div
      class="statementis-row"
    >
      <div
        class="statementis-col-lg-12"
      >
        <h2
          class="st_title"
        >
          <i
            class="uil uil-file-alt"
          />
           Statements
        </h2>
      </div>
    </div>
    <div
      class="statementis-row"
    >
      <div
        class="statementis-col-lg-8"
      >
        <div
          class="top_countries mt-30"
        >
          <div
            class="top_countries_title"
          >
            <h2>
              Earnings
            </h2>
          </div>
          <div
            class="statement_content"
          >
            <p
              class="tt-body"
            >
              If you are not an instrutor, you can't use this section.
            </p>
            <div>
              <table
                class="statement-summary__table"
              >
                <thead>
                  <tr>
                    <th>
                      <p
                        class="t-heading"
                      >
                        My funds
                      </p>
                    </th>
                    <th>
                      <p
                        class="t-heading"
                      >
                        Earnings
                      </p>
                    </th>
                    <th>
                      <p
                        class="t-heading"
                      >
                        Cursus Fees
                      </p>
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td
                      class="statement-summary__funds"
                    >
                      <p
                        class="js-earnings__instructor-funds-wrapper"
                      >
                        <span
                          class=""
                        />
                        <span
                          class="js-earnngs__instructor-funds t-currency"
                        >
                          $289.64
                        </span>
                      </p>
                    </td>
                    <td
                      class="statement-summary__earnings"
                    >
                      <p
                        class="js-earnings__earnings-wrapper"
                      >
                        <span
                          class="tt__earning"
                        >
                          +
                        </span>
                        <span
                          class="js-earnings__earnings t-currency"
                        >
                          $458.00
                        </span>
                      </p>
                    </td>
                    <td
                      class="statement-summary__fees"
                    >
                      <p
                        class="js-earnings__fees-wrapper"
                      >
                        <span
                          class="tt__earning"
                        >
                          -
                        </span>
                        <span
                          class="js-earnings__fees t-currency"
                        >
                          $154.86
                        </span>
                      </p>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
      <div
        class="statementis-col-lg-4"
      >
        <div
          class="top_countries mt-30"
        >
          <div
            class="top_countries_title"
          >
            <h2>
              View Invoices
            </h2>
          </div>
          <div
            class="statement_invoice_content"
          >
            <div
              class="date_selector"
            >
              <div
                class="ui selection"
                tabindex="0"
              >
                <input
                  name="date"
                  type="hidden"
                  value="Monthly Invoices"
                />
                <div
                  class="sis-drop"
                >
                  <span
                    class="section-title-wrapper"
                  >
                    <span
                      class="section-title-text"
                    >
                      Monthly Invoices
                    </span>
                    <img
                      alt="Dropdown Icon"
                      class="sidebar-dropdown-caret-icon"
                      src="dropdown.png"
                    />
                  </span>
                </div>
              </div>
              <button
                class="st_download_btn"
              >
                <i
                  class="uil uil-download-alt"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="statementis-col-lg-12"
      >
        <div
          class="more_options_tt"
        >
          <li>
            <div>
              <button
                class="more_items_14 active"
              >
                This Month
              </button>
            </div>
          </li>
          <li>
            <div>
              <button
                class="more_items_14"
              >
                Last Month
              </button>
            </div>
          </li>
          <li>
            <div
              class="explore_search"
            >
              <div
                class="sis-search-box"
              >
                <div
                  class="sis-box-iput"
                >
                  <input
                    class="sis-search"
                    placeholder="Document Number"
                    type="text"
                  />
                  <i
                    class="uil uil-search-alt icon icon8"
                  />
                </div>
              </div>
            </div>
          </li>
        </div>
      </div>
    </div>
    <div
      class="MyCourse-tab-pane active"
      id="my-courses-tab"
      role="tabpanel"
    >
      <div
        class="MyCouser-containerr"
      >
        <div
          class="MyCouser-table-container"
        >
          <table
            class="MyCourse-table ucp-table"
          >
            <thead
              class="MyCourse-thead-s"
            >
              <tr>
                <th
                  class="MyCourse-text-center"
                  scope="col"
                >
                  Date
                </th>
                <th
                  class="MyCourse-text-center"
                >
                  Order ID
                </th>
                <th
                  class="MyCourse-text-center"
                  scope="col"
                >
                  Type
                </th>
                <th
                  class="MyCourse-text-center"
                  scope="col"
                >
                  Title
                </th>
                <th
                  class="MyCourse-text-center"
                  scope="col"
                >
                  Amount
                </th>
                <th
                  class="MyCourse-text-center"
                  scope="col"
                >
                  Fees
                </th>
                <th
                  class="MyCourse-text-center"
                  scope="col"
                >
                  Invoice
                </th>
              </tr>
            </thead>
            <tbody />
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
